### ISSUE TITLE
[Art competition] Invariant

### ISSUE THREAD

**Issue by sujin0529** [prosyslab-classroom/cs524-program-analysis#101]
<img src=https://user-images.githubusercontent.com/44044134/204132035-5d58b353-d183-4951-99be-56b66c773324.png height=500>

## Short Description
For me, [Invariant] means [burning the stone with a torch]".

## Long Description (up to 3 sentences)
Stone does not melt by fire. The invariant is a state that does not change even if you continue to execute statements in iteration. So, I thought that a stone that does not melt is similar to the concept of the invariant.

**Comment by KihongHeo** [comment]
Here what is "iteration" or "execution? Firing?

**Comment by sujin0529** [comment]
Yes! I thought that "iteration" would be like continuing to be burned by a torch.
